warehouse/internal/repo/upload.go

```diff
@@ -561,27 +561,31 @@ func (u *Uploads) PendingTableUploads(ctx context.Context, namespace string, upl
 		ON
 			UT.id = TU.wh_upload_id
 		WHERE
-		  	UT.id <= $1 AND
-			UT.destination_id = $2 AND
-		   	UT.namespace = $3 AND
+			UT.destination_id = $1 AND
```

Comment:
We can add a comment that the logic here should be in sync with the other method